addons/mail/static/src/core/common/composer.js

```diff
@@ -87,6 +87,7 @@ export class Composer extends Component {
     setup() {
         super.setup();
         this.isMobileOS = isMobileOS();
+        this.isIosApp = isIosApp();
```

Comment:
`isIosApp` only applies to the iOS mobile app.
The bug is also there so it's good, but the mobile app is deprecated and we encourage PWA instead.
`isIosApp` does not take the PWA into account.

To take the PWA on iOS, you need both `isDisplayStandalone()` (PWA) and `iOS()`.

I would name it like `this.isIosPwa` for clarity (even though acronyms look weird with camelCase, this is still the best for readability I've found without being too verbose).